core/tests/protractor_utils/ExplorationEditorMainTab.js

```diff
@@ -615,81 +653,75 @@ var ExplorationEditorMainTab = function() {
     }
   };
 
+  /**
+   * Parse for Angular selectors and remove them.
+   * @param {string} [ruleDescription] - Interaction type.
+   * @param {string|string[]} [providedText] - Feedback text to replace with.
```

Comment:
Have deterministic types -- either string or Array<string> but not both.

Reply:
Done!